_includes/commands/advtable-cmds.md

```diff
@@ -24,4 +18,23 @@ tinymce.activeEditor.execCommand('mceSortTableByColumnAsc')
 tinymce.activeEditor.execCommand('mceSortTableByColumnDesc')
 tinymce.activeEditor.execCommand('mceSortTableByRowAsc')
 tinymce.activeEditor.execCommand('mceSortTableByRowDesc')
-```
\ No newline at end of file
+tinymce.activeEditor.execCommand('mceTableToggleSeries', false, { name: 'numeric', location: 'left' })
+```
+
+### `mceSortTableAdvanced`
+
+`mceSortTableAdvanced` accepts an object with the following key-value pairs:
+
+- `sortby` - `'row'` or `'column'`
+- `roworcol` - a zero-indexed integer in a string representing the row from the top of the table or column from the left of the table.
+- `sort` - `'row'`, `'column'`, `'selection'`, or `'table'`
+- `order` - `'ascending'` or `'descending'`
+
+### `mceTableToggleSeries`
+
+`mceTableToggleSeries` accepts an object with the following key-value pairs:
+
+| Name | Value | Requirement | Description |
+| ---- | ----- | ----------- | ----------- |
+| name | `string` | Required |  Specifies the series to toggle. Series and their associated names are configured using the [advtable_value_series option]({{site.baseurl}}/plugins/premium/advtable/#advtable_value_series). If the series name provided is not present on the table, a series column will be inserted or an existing series column will be replaced by the new series. |
+| location | `'left'` | Required | Specifies the location of where the series is toggled in the table. For example, a location of `left` will toggle a series column on the left side of the table.  |
```

Comment:
Should this be "`'left'` or `'right'`"

Reply:
Actually should just even be included at all? This is just something we put as a placeholder and definitely isn't required. As such, I believe this should be removed from the docs for now.

Reply:
That's a good point. I will remove `location` which should also make it less confusing.